assets/option/option_test_api.yml

```
@@ -25,4 +25,10 @@ plugins:
     - plugin: client_proxy
       kwargs:
         proxy_client_key: photo_concurrent_fetcher_proxy
-        whitelist: [ api, ]
\ No newline at end of file
+        whitelist: [ api, ]
+
+    - plugin: advanced-retry
+      kwargs:
+        retry_config:
+          retry_rounds: 3 # 一共对域名列表重试3轮
+          retry_domain_max_times: 5 # 当一个域名重试次数超过5次，忽略该域名，不再重试
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)